client/gutenberg/extensions/mailchimp/edit.jsx

```diff
@@ -186,7 +186,12 @@ class MailchimpSubscribeEdit extends Component {
 			<div className={ className }>
 				{ ! audition && (
 					<form ref={ this.formRef }>
-						<TextControl placeholder={ emailPlaceholder } onChange={ () => false } type="email" />
+						<TextControl
+							disabled
+							placeholder={ emailPlaceholder }
+							onChange={ () => false }
+							type="email"
```

Comment:
Would it be worth adding a title in case users are attempting to directly manipulate the placeholder? Direct manipulation of the placeholder may be confusing, I assume that's why we moved it to the sidebar, but the lack of direct manipulation may also be confusing 🙂 

@thomasguillot What do you think?

```suggestion
							title={ __( 'You can edit the email placeholder in the sidebar.' ) }
							type="email"
```

Reply:
Adding a title is a great idea 😊

Reply:
Good idea. Done in https://github.com/Automattic/wp-calypso/pull/30751/commits/cdb1b738d00a205fb37eb15d6420f9f466a95ec1. @thomasguillot am I still good to land this?

Reply:
Yes 👍